policy.md

```diff
@@ -226,24 +320,63 @@ Each audit statement MUST be accompanied by documentation of the audit team qual
 * How the audit team and team members are bound by law, regulation or professional standards to render an independent assessment of the CA (e.g., https://pub.aicpa.org/codeofconduct/Ethics.aspx# 0.300.050 Objectivity and Independence; CPA Canada, Rule 204; or Annex A of ETSI EN 319 403/403-1, respectively); and
 * Whether the audit team relied on any third-party specialists or affiliate audit firms, and if so, their names and where they performed services.
 
-### 5.3 Annual CCADB Self-Assessments ###
+## 6. Additional Required Practices
+
+### 6.1 Disclosing and Responding to Incidents
+
+The CCADB [Incident Reporting Guidelines (IRGs)](https://www.ccadb.org/cas/incident-report) intend to present a common format for publicly disclosing incidents. CA Owners who are required to publicly disclose incidents MUST adhere to the CCADB IRGs. Those CA Owners MUST update the incidents and respond to questions in accordance with the IRGs until closure.
```

Comment:
I think it would be reasonable to read the last sentence here as saying that, if the IRGs are updated in the middle of the incident process (i.e. after the report has been filed, but before it has been closed), then the report itself needs to be updated to be in accordance with the new IRGs. I don't think that's the intent (I think this just means that you need to update the bug every 7ish days), so maybe this should be rephrased a little.

Reply:
Agree, they would not need to attempt to update past content, but we would expect any subsequent updates SHOULD align with the new IRGs, but MUST align with the new IRGs if superseded by more than 30 days. We could rephrase the statement to:

```suggestion
The CCADB [Incident Reporting Guidelines (IRGs)](https://www.ccadb.org/cas/incident-report) intend to present a common format for publicly disclosing incidents. CA Owners who are required to publicly disclose incidents:
- SHOULD always use the latest available version of the CCADB IRG templates, and
- MUST NOT use a version of an IRG template that has been superseded by more than 30 calendar days before the corresponding report (i.e., preliminary, full, or closure) is posted.
```